net-mail/poppassd_ceti/poppassd_ceti-1.8.7.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pam
+
+# Long ago it was just "poppassd", but upstream now seems to have
+# settled on "poppassd-ceti" (instead of "poppassd_ceti" or no suffix).
+MY_PN="poppassd-ceti"
+MY_P="${MY_PN}-${PV}"
+S=${WORKDIR}/${MY_P}
+
+DESCRIPTION="Password change daemon with PAM support"
+HOMEPAGE="https://github.com/kravietz/poppassd-ceti"
+SRC_URI="https://github.com/kravietz/${MY_PN}/releases/download/v${PV}/${MY_P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="sys-libs/pam"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/doc-sudoAlternative-1.8.7.patch"
+)
+
+src_install() {
+	dodoc README.md
+
+	pamd_mimic_system poppassd auth account password
+
+	insinto /etc/xinetd.d
+	newins "${FILESDIR}"/poppassd.xinetd poppassd
+
+	insinto /usr/sbin
+	insopts -o root -g bin -m 500
+	doins poppassd
+}
+
+pkg_postinst() {
+	elog "/usr/sbin/poppassd has to be run as root to work."
```

Comment:
This also could fit readme.gentoo thingie.

Reply:
I've made all these changes, although I have a couple of possible
concerns about the README.gentoo changes:

   1. The README.md patch has already been submitted upstream and
      accepted into its source control, although it hasn't been
      officially released.

      (Generally, the "is xinetd a dependency" question is
      more general than just Gentoo.)

   2. The elog'ed output (and README.md patch) was a replacement for a
      dependency on xinetd (either an "unconditional but
      not-strictly-necessary RDEPEND", or a "USE-controlled RDEPEND").

      I wrote up an extensive analysis about various options,
      inconsistencies, etc in how xinetd dependencies are handled
      (or should be handled) on Oct 25 above.  (Short version:
      xinetd is essentially NEVER the only way to run any mini-service,
      even in cases where it is the most common, so is it a
      dependency or not?)

      Is README.gentoo your recommended way to handle xinetd
      semi-dependencies?

Reply:
If it's been accepted upstream, then the patch is fine -- but please include the original commit message (i.e. get it from git-format patch) or at least upstream commit link.

I'd personally still go for readme.gentoo to avoid extra files. Generally, the idea is that if you have a lot of postinst messages and usually they need to be shown to the user only once, readme.gentoo is the solution. Then the user sees them only once, and can read them from README.gentoo file afterwards.

Reply:
I just pushed up another rebase:
  - GPL-2+ (The license is somewhat ambiguous in 1.8.7, but I agree that GPL-2+ is reasonable, and it may prevent needing to change it again in a future bump.)
  - Use readme.gentoo-r1.eclass (For my earlier push today I only pushed a README.gentoo file to install directly; I hadn't realized that you were talking about an eclass until you started talking about extra files and "shown ... only once".  I like the eclass a lot better.)
  - Using doexe, etc (I was just copying the old ebuild and didn't realize there was a better alternative).
  - Not bothering with the README.md patch, since I don't really need to duplicate the sudo information that I'm also describing in the readme.gentoo information.  (It won't hurt in a future bump, though.)